configure.py

```diff
@@ -707,10 +713,11 @@ def get_native_cuda_compute_capabilities(environ_cp):
   """
   device_query_bin = os.path.join(
       environ_cp.get('CUDA_TOOLKIT_PATH'), 'extras/demo_suite/deviceQuery')
-  cmd = (r'"%s" | grep "Capability" | grep -o "[0-9]*\.[0-9]*" | sed '
-         '":a;{N;s/\\n/,/};ba"') % device_query_bin
   try:
-    output = run_shell(cmd)
+    output = run_shell(device_query_bin).split('\n')
```

Comment:
Looks like it is not happy that it cannot find deviceQuery. https://ci.tensorflow.org/job/tensorflow-pull-requests-gpu/6417/console
Can we add a check for that if we are not using shell=True?